sap/5.7/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -86,6 +87,40 @@ Relies on an X509 certificate to authenticate the user.
 | Reconnection a| <<Reconnection>> |  Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
 |===
 
+[[sap_kerberos]]
+=== Kerberos
+
+Relies on an X509 certificate to authenticate the user.
```

Comment:
This connection provider implements the Kerberos Protocol to authenticate and authorise the user.